src/daemon/utils_ssl.c

```diff
@@ -0,0 +1,104 @@
+/**
+ * collectd - src/utils_ssl.c
+ * Copyright (C) 2007-2010  Florian octo Forster
```

Comment:
Ok, I can change but my goal is to help collectd developers to improve this wonderful tool,